File: src/main/java/myLang/block/print/PrintContext.java
package myLang.block.print;

import myLang.ContextHandler;
import myLang.response.MyLangException;
import myLang.response.Response;
import myLangParser.MyLangParser;

public class PrintContext implements ContextHandler<MyLangParser.PrintContext> {

    @Override
    public Response handler(MyLangParser.PrintContext ctx) throws MyLangException {
        return defaultHandler(ctx, 1);
    }
}

File: src/main/kotlin/myLangKotlin/block/print/PrintContext.kt
package myLangKotlin.block.print

import myLangKotlin.ContextHandler
import myLangKotlin.response.MyLangException
import myLangKotlin.response.Response
import myLangParser.MyLangParser

class PrintContext : ContextHandler<MyLangParser.PrintContext> {

    @Throws(MyLangException::class)
    override fun handler(ctx: MyLangParser.PrintContext): Response<*> {
        return defaultHandler(ctx, 1)
    }
}
